go/ql/lib/semmle/go/frameworks/stdlib/Os.qll

```diff
@@ -43,4 +43,14 @@ module Os {
       input = inp and output = outp
     }
   }
+
+  private class Stdin extends SourceNode {
+    Stdin() {
+      exists(Variable osStdin | osStdin.hasQualifiedName("os", "Stdin") |
+        this.asExpr() = osStdin.getARead().asExpr()
```

Comment:
```suggestion
        this = osStdin.getARead()
```
There are two kinds of read instructions which aren't expressions. Neither seems particularly relevant here, but it seems easier to not restrict to expressions.